src/notation/internal/notationinteraction.cpp

```diff
@@ -481,7 +482,7 @@ void NotationInteraction::moveChordNoteSelection(MoveDirection d)
     notifyAboutSelectionChanged();
 }
 
-void NotationInteraction::select(const std::vector<Element*>& elements, SelectType type, int staffIndex)
+void NotationInteraction::selectInternal(const std::vector<Element*>& elements, SelectType type, int staffIndex)
```

Comment:
Just a suggestion. I usually in such cases call `doSomeThing` - that is, direct action. In this case `doSelect`. Like this in many places in the code.